catkin_ws/src/kpr_arm_control/config/controllers.yaml

```diff
@@ -0,0 +1,11 @@
+controller_list:
+  - name: ""
+    action_ns: follow_joint_trajectory
+    type: FollowJointTrajectory
+    joints:
+      - shoulder_pan_joint
+      - shoulder_lift_joint
+      - elbow_joint
+      - wrist_1_joint
+      - wrist_2_joint
+      - wrist_3_joint
```

Comment:
Additionally the directory should be called cfg/ in order to follow the ROS conventions